extensions/sql-database-projects/resources/templates/newSdkSqlProjectTemplate.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
```

Comment:
I think we can use `ToolsVersion="Current"` here
https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-toolset-toolsversion?view=vs-2022

Reply:
Actually maybe the ToolsVersion and xmlns isn't even needed. The sample SDK sqlproj in the DacFx repo doesn't have them and looks like SDK style csprojs also don't have them.